funcx_endpoint/funcx_endpoint/endpoint/endpoint_manager.py

```diff
@@ -22,7 +22,6 @@
 from globus_sdk import GlobusAPIError, NetworkError
 
 import funcx_endpoint
-from funcx.utils.errors import *
```

Comment:
Did you check if anything from errors was used in this file?

Reply:
Because `flake8` would now report unknown names, it shouldn't be possible.

When I checked, I saw use of `Exception`, `OSError`, `FuncxResponseError` (imported on the line below), and `GlobusAPIError` and `NetworkError` from the `globus_sdk`.
I didn't see any use of exception classes from this import.